cmd/testing/basetls.go

```diff
@@ -66,7 +66,7 @@ func (s *BaseTLSSuite) SetupTest() {
 	s.Host = s.Node.APIServer.Address //NOTE: 0.0.0.0 will not work because we're testing TLS validation
 	s.Port = s.Node.APIServer.Port
 	s.Client = client.NewAPIClient(client.LegacyTLSSupport{UseTLS: true, Insecure: false}, s.Host, s.Port)
-	s.ClientV2 = clientv2.New(fmt.Sprintf("http://%s:%d", s.Host, s.Port), clientv2.WithTLS(true))
+	s.ClientV2 = clientv2.New(fmt.Sprintf("http://%s:%d", s.Host, s.Port), clientv2.WithTLS(true), clientv2.WithInsecureTLS(true))
```

Comment:
```suggestion
	s.ClientV2 = clientv2.New(fmt.Sprintf("http://%s:%d", s.Host, s.Port))
```

still haven't removed the TLS stuff here...

Reply:
Is this the right change? The v1 client on the line above has `UseTLS: true`

Reply:
Oops, totally missed that this was the basetls suite. My bad.